### PR TITLE
WAY FUCKED new branch 

### PR DESCRIPTION
**Pull request by Militaires**
- [x] port all existing stands to new system

- [x] create new more exciting stand types

- [x] allow stands to use the new system

- [ ] implement a tool to summon the stand using the new system

- [ ] come back and add even more types

### CODE REVIEW

**Comment by Militaires** [comment]
shit fucked with martial.dm will revert that

**Comment by Militaires** [comment]
wow this is really fucked let me reset and re-add my files

File: code/modules/mob/living/simple_animal/guardian/guardian.dm [hunk 1]
```diff
@@ -469,39 +323,28 @@ var/global/list/parasites = list() //all currently existing/living guardians
...
-	for(var/datum/guardian_abilities/I in current_abilities)
-		I.recall_act()
 	return TRUE
 
 /mob/living/simple_animal/hostile/guardian/proc/ToggleMode()
-	if(has_mode)
-		for(var/datum/guardian_abilities/I in current_abilities)
-			I.handle_mode()
-	else
-		src << "<span class='danger'><B>You don't have another mode!</span></B>"
```
Comment (Iamgoofball) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: you dont get to bring friends

**Comment by kevinz000** [comment]
what the fuck are you doing to my flightsuit code you fucking nerd reee

**Comment by Iatots** [comment]
What is this new system?

Does it let short-ranged stands manifest on the user as piggybackers?
Does it let users apply stand effects themselves at range 0 with just a stand's arm showing?

**Comment by Militaires** [comment]
@iatots ill answer all of that and more after i unfuck it which is soon my friend

**Pull request closed without merging**